kzz7yh-a23210/cod-ve07v1_kzz7yh-a23210.xml

```diff
@@ -101,8 +102,8 @@
       perfe<lb ed="#V" n="92" break="no"/>ctum. ergo perfectio diuini esse non requirit quod in deo sit generatio. 
     </p>
                <p xml:id="kzz7yh-a23210-d1e180">
-                  <lb ed="#V" n="93"/>Contra Psat in persona patrus loquens ad filium ait. 
-      <lb ed="#V" n="94"/>Ege piode genuriem: eedverbum 
+                  <lb ed="#V" n="93"/>Contra Psalmum in persona patris loquens ad filium ait. 
```

Comment:
I think Psalmus is the subject of ait here (Psalmista might even make more sense)
```suggestion
                  <lb ed="#V" n="93"/>Contra Psalmus in persona patris loquens ad filium ait. 
```